internal/featuresupport/features_misc.go

```diff
@@ -300,3 +300,57 @@ func (f *skipMcoReboot) getFeatureActiveLevel(cluster *common.Cluster, infraEnv
 	}
 	return activeLevelActive
 }
+
+// Non-standard HA OCP Control Plane
+type NonStandardHAControlPlane struct{}
+
+func (f *NonStandardHAControlPlane) New() SupportLevelFeature {
+	return &NonStandardHAControlPlane{}
+}
+
+func (f *NonStandardHAControlPlane) getId() models.FeatureSupportLevelID {
+	return models.FeatureSupportLevelIDNONSTANDARDHACONTROLPLANE
+}
+
+func (f *NonStandardHAControlPlane) GetName() string {
+	return "Non-standard HA OCP Control Plane"
+}
+
+func (f *NonStandardHAControlPlane) getSupportLevel(filters SupportLevelFilters) models.SupportLevel {
+	supported, err := common.BaseVersionGreaterOrEqual(common.MinimumVersionForNonStandardHAOCPControlPlane, filters.OpenshiftVersion)
+	if !supported || err != nil {
+		return models.SupportLevelUnavailable
+	}
+
+	if filters.PlatformType != nil && *filters.PlatformType != models.PlatformTypeBaremetal {
```

Comment:
not supported in None?

Reply:
This is what I saw in https://docs.google.com/presentation/d/1eMz3ON5y1Rk2QI2zH-QUGh5oKoMyOHZrCSSZ8aHQQPQ/edit#slide=id.g26e42bb9faa_0_197

Reply:
Probably can work with none as well

Reply:
Just not nutanix, vspere, etc